Backend/src/main/java/ca/mcgill/cooperator/controller/CourseOfferingController.java

```diff
@@ -28,16 +31,45 @@ public CourseOfferingDto getCourseOfferingById(@PathVariable int id) {
         return ControllerUtils.convertToDto(courseOfferingService.getCourseOfferingById(id));
     }
 
+    @GetMapping("")
+    public List<CourseOfferingDto> getCourseOfferings() {
+        return ControllerUtils.convertCourseOfferingListToDto(
+                courseOfferingService.getAllCourseOfferings());
+    }
+
+    @GetMapping("/course/{id}")
```

Comment:
i dont think you need /course here, just /{id} works

Reply:
i need the /course/ bc otherwise its indistinguishable from get course offering by id route